commands/init_test.go

```diff
@@ -1,34 +0,0 @@
-package commands
-
-import (
-	"context"
-	"fmt"
-	"os/exec"
-	"strconv"
-	"testing"
-
-	th "github.com/filecoin-project/go-filecoin/testhelpers"
-	"github.com/stretchr/testify/require"
-)
-
-func TestDownloadGenesis(t *testing.T) {
-	ctx, cancel := context.WithCancel(context.Background())
-	defer cancel()
-
-	port, err := th.GetFreePort()
-	require.NoError(t, err)
-
-	err = exec.CommandContext(
-		ctx,
-		th.ProjectRoot("tools/genesis-file-server/genesis-file-server"),
-		"--genesis-file-path",
-		th.ProjectRoot("fixtures/genesis.car"),
-		"--port",
-		strconv.Itoa(port),
-	).Start()
-	require.NoError(t, err)
-
-	td := th.NewDaemon(t, th.GenesisFile(fmt.Sprintf("http://127.0.0.1:%d/genesis.car", port))).Start()
```

Comment:
this entire test was moved to `init_daemon_test.go` because its a **daemon** test.